androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/PlayerControls.kt

```diff
@@ -0,0 +1,86 @@
+package com.profusion.androidenhancedvideoplayer.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.profusion.androidenhancedvideoplayer.R
+
+@Composable
+fun PlayerControls(
+    isPlaying: Boolean,
+    onPreviousClick: () -> Unit,
+    onPauseToggle: () -> Unit,
+    onNextClick: () -> Unit,
+    decoration: ControlsDecoration,
```

Comment:
I don't think it's much worth it to abstract those into a data class. IMO, better to have more arguments with default behavior explicitly on the function call - less redirects to documentation when one wants to customize.

Reply:
Also, I'd expand this argument
```suggestion
    previousIconContent: @Composable () -> Unit = { DefaultPreviousIcon() },
    playOrPauseIconContent: @Composable (Boolean) -> Unit = { isPlaying ->
        DefaultPlayOrPauseIcon(
            isPlaying
        )
    },
    nextIconContent: @Composable () -> Unit = { DefaultNextIcon() }
```